courses/fundamentals_of_ada/quiz/mutable_with_array/prj.gpr

```diff
@@ -0,0 +1,13 @@
+project Prj is
```

Comment:
Are any of these GPR / ADC files different? If not, can't we just put one at the "quiz" level and use "../prj.gpr" when building?

Reply:
I don't see what you mean, it does not seem possible unless we add e.g. scenario variables to the mix, do you have a ready-made example?

Reply:
If the CI to build/test quizzes is the same across all courses, then your way is OK. But, if the CI is specific to fundamentals, and the GPR file is common, couldn't we point to the common GPR file rather than making copies everywhere?